homeassistant/components/manual/alarm_control_panel.py

```diff
@@ -74,15 +73,16 @@
 
 def _state_validator(config):
     """Validate the state."""
-    config = copy.deepcopy(config)
```

Comment:
The deepcopy was added here to make tests pass: https://github.com/home-assistant/core/pull/9264/commits/5dc624f57455bd89c5b63369fcc3478fb2896509#r137995580

I think the problem was mutating nested dictionaries, this PR now copies the nested dictionaries instead of mutating them.
However, tests pass also without the copying, so I'm not sure if we still need it?